tests/test_cli_omo_tmux_respawn_pane.py

```
@@ -0,0 +1,489 @@
+        fake_home,
+        [
+            "__tmux-compat",
+            "respawn-pane",
+            "-t",
+            subagent_pane_token,
+            ATTACH_COMMAND,
+        ],
+    )
+    assert_failure(non_forced_respawn, "OMO non-forced respawn-pane", "requires -k")
+    if state.respawn_params:
+        raise AssertionError(f"non-forced respawn must not call surface.respawn: {state.respawn_params!r}")
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)